### PR TITLE
feat(FormattingPatterns): add timestamp

### PR DESCRIPTION
**Pull request by ckohen**
**Please describe the changes this PR makes and why it should be merged:**

Timezone formatted timestamps!!!!! (but the regex matches)

**Reference Discord API Docs PRs or commits:**
https://github.com/discord/discord-api-docs/pull/3135
https://github.com/discord/discord-api-docs/pull/3141

### CODE REVIEW

File: globals.ts [hunk 1]
```diff
@@ -61,6 +61,24 @@ export const FormattingPatterns = {
 	 * The `name` and `id` group properties are present on the `exec` result of this expression
 	 */
 	StaticEmoji: /<:(?<name>\w{2,32}):(?<id>\d{17,20})>/,
+	/**
+	 * Regular expression for matching a timestamp, either unflagged or flagged
+	 *
+	 * The `timetsamp` and `flag` group properties are present on the `exec` result of this expression
```
Comment (vladfrangu) [inline globals.ts:67]: Typo'd

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: globals.ts [hunk 2]
```diff
@@ -61,6 +61,24 @@ export const FormattingPatterns = {
...
+	/**
+	 * Regular expression for matching strictly unflagged timestamps
+	 *
+	 * The `timestamp` group property is present on the `exec` result of this expression
+	 */
+	UnflaggedTimestamp: /<t:(?<timestamp>-?\d{1,13})>/,
+	/**
+	 * Regular expression for matching strictly flagged timestamps
+	 *
+	 * The `timestamp` and `fllag` group properties are present on the `exec` result of this expression
```
Comment (kyranet) [inline globals.ts:79]:
```suggestion
	 * The `timestamp` and `flag` group properties are present on the `exec` result of this expression
```

**Review by kyranet — CHANGES_REQUESTED** [review]
Also, should it be named `flag` and not `style` or `pattern`? Is this the name used by Discord?

**Comment by advaith1** [comment]
Discord calls them flags: [docs](https://github.com/discord/discord-api-docs/pull/3135/files?short_path=9c04655#diff-9c0465597400a1bfff16968024a03ed3e42f9f35b43b9d4419bb0156d47ff9d2)

**Comment by devsnek** [comment]
style would probably be a better name, feel free to change the docs lol

**Review by kyranet — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**